File: app/src/main/java/mx/edu/upqroo/kristenandroid/widget/ListViewWidgetService.java
package mx.edu.upqroo.kristenandroid.widget;

import android.content.Intent;
import android.widget.RemoteViewsService;

public class ListViewWidgetService extends RemoteViewsService {

    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new ListViewRemoteViewsFactory(getApplicationContext(), intent);
    }
}

File: app/src/main/java/mx/edu/upqroo/kristenandroid/widget/ListViewWidgetService.kt
package mx.edu.upqroo.kristenandroid.widget

import android.content.Intent
import android.widget.RemoteViewsService

class ListViewWidgetService : RemoteViewsService() {
    override fun onGetViewFactory(intent: Intent): RemoteViewsFactory {
        return ListViewRemoteViewsFactory(applicationContext)
    }
}
